src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,51 @@
-export const PeopleFilters = () => {
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import React from 'react';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+interface Props {
+  searchParams: URLSearchParams
+  centuries: string[]
+  query: string
+  sex: string
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+export const PeopleFilters: React.FC<Props> = ({
+  searchParams,
+  centuries,
+  query,
+  sex,
+  onQueryChange,
+}) => {
+  const centuriesArray = ['16', '17', '18', '19', '20'];
```

Comment:
Instead of hardcoding the centuriesArray, it's better to pass it as a prop or import it from a constants file.